crates/fluvio-connector-derive/tests/mod.rs

```diff
@@ -0,0 +1,8 @@
+#[test]
+fn ui() {
+    let t = trybuild::TestCases::new();
+    t.compile_fail("tests/ui/sink.rs");
+    t.compile_fail("tests/ui/wrong_direction.rs");
+    t.compile_fail("tests/ui/wrong_input_len.rs");
+    t.compile_fail("tests/ui/not_async.rs");
```

Comment:
We should add some `compile_success`'s to this. I wish there was a version of this that had the expanded rust.

Reply:
`compile_success` beside the compilation also runs the `main` method that, in our case, won't succeed due to required input parameters. I assume that is why `https://github.com/infinyon/trybuild/commits/check_option` exists. But it's already outdated. With that version, the code doesn't compile. We need to rebase our fork, but it might affect other tests, so it should be done separately. I will try to do this on the next iteration when I work on supporting sink connectors here.

The expanded version is too verbose and will often change to be stored somewhere static.